examples/step-48/step-48.cc

```diff
@@ -58,7 +58,7 @@ namespace Step48
   // programs), whereas the degree of the finite element is more crucial, as
   // it is passed as a template argument to the implementation of the
   // Sine-Gordon operator. Therefore, it needs to be a compile-time constant.
-  const unsigned int dimension = 2;
+  const unsigned int dimension = 3;
```

Comment:
On purpose? What is the run time of the program with this? We like to keep the default quick.

Reply:
This is definitely a mistake. I played around with it once too often. :wink: